### PR TITLE
Upgrade to v2.7.2 of the Packet Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: sdk/nodejs/tsconfig.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
         "billingCycle.ts",
         "config/index.ts",
         "config/vars.ts",
-        "connect.ts",
```
Comment (mikhailshilkov) [inline sdk/nodejs/tsconfig.json:20]: Do we need to delete the files? Is that a breaking change?

Comment (stack72) [reply sdk/nodejs/tsconfig.json:20]: This was autogenerated... I am sure this is related to https://github.com/terraform-providers/terraform-provider-packet/blob/master/CHANGELOG.md#270-november-19-2019

Comment (mikhailshilkov) [reply sdk/nodejs/tsconfig.json:20]: Why are `connect.ts` and friends still there?

Comment (stack72) [reply sdk/nodejs/tsconfig.json:20]: The last release we made of the provider was 2.6.1 when the old resource was deprecated

Comment (stack72) [reply sdk/nodejs/tsconfig.json:20]: also, it doesn't seem that our generation cleans out old files as part of the SDK generation

Comment (mikhailshilkov) [reply sdk/nodejs/tsconfig.json:20]: So what's the normal action then? Delete it manually? Or let them pile on ~forever?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-* Upgrade to v2.7.1 of the Packet Terraform Provider
+* Upgrade to v2.7.2 of the Packet Terraform Provider
```
Comment (mikhailshilkov) [inline CHANGELOG.md:5]: Should we explicitly mention the deleted resource?

**Comment by stack72** [comment]
I will remove the files and then add a breaking change line to the changelog for this PR @mikhailshilkov

**Review by mikhailshilkov — APPROVED** [review]
Thanks!

**Pull request merged**